server/app/services/mongo_pubsub.rb

```diff
@@ -45,7 +48,7 @@ def queue_message(data)
 
     def process
       @process = true
-      Celluloid::Future.new {
+      async_thread do
```

Comment:
This `MongoPubsub::Subscriber` needs some actual error handling, because if the `MongoPubsub#subscribe do ... end` block raises an error, then the process thread will die, leaving the `Subscription` in the `@process = true` state. The `MongoPubsub#tail!` actor thread will continue calling `subscription.queue_message` to push messages to the `@queue`, but there won't be anyone there to read them.